goto/commands/add.py

```diff
@@ -20,4 +21,4 @@ def add(magic, args):
     magic.add_shortcut(magicword, uri)
     magic.save()
 
-    return u'Added magic word %s' % magicword, None
+    return 'Added magic word {}'.format(magicword), None
```

Comment:
I think the output here should just be `return magicword, None`

Reply:
Would love to hear what your thought is here :) 

I think I have too much design values and principles and guidelines for this project in my head. 
It would be valuable for further development if I made them more transparent and wrote them down. 

I will work on that from Warszawa. 

The reasoning behind this output is - this is the first thing new users will want to try. And the whole concept of goto is very new, and mysterious at this point. Nielsens principle of showing the user information about what the system does in a understandable language is therefore applied  - in order to try to achieve a good learning experience. 

I think there is value to beeing a bit more explicit here and writing out 'Added magic word'. 


This must be seen in relation to my comments on the output of using goto <magicword> , and wanting not to show output by default. This may seem a bit contradicting to the logic above. At first.  However there is some reasoning behind this also:  The user gets clear feedback from the system on the effect of the command -- the browser is opening, it is focused, and you see that it is on the url you were opening. 


This is analogous to the unix commands `cd` `kill` `mkdir` `mv` etc.  They all are silent on success, and verbose on faults.

Reply:
> This is analogous to the unix commands `cd` `kill` `mkdir` `mv` etc. They all are silent on success, and verbose on faults.

Good point! I no longer think just returning the name of the magicword is a good idea.